frontend/src/ManageUsersPage/UserList.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { graphql } from 'react-apollo';
+import gql from 'graphql-tag';
+import Table from './Table';
+
+const manageUsersHeaders = [{
```

Comment:
This is where I'm giving options to headers on whether they will be sortable and filterable via search. _Yet to be implemented fully_